### PR TITLE
avdl autoreset ui rework

### PR DESCRIPTION
**Pull request by pzduniak**

### CODE REVIEW

File: go/engine/account_reset.go [hunk 1]
```diff
@@ -239,11 +250,19 @@ func (e *AccountReset) resetPrompt(mctx libkb.MetaContext, status *accountResetS
 		}
 		if response == keybase1.ResetPromptResponse_NOTHING {
 			// noop
-			mctx.Info("Reset not completed.")
+			if err := mctx.UIs().LoginUI.DisplayResetMessage(mctx.Ctx(), keybase1.DisplayResetMessageArg{
```
Comment (jakob223) [inline go/engine/account_reset.go]: instead of `if err return err else return nil`, just `return mctx.UIs().....`

**Comment by zapu** [comment]
cc @patrickxb on UI changes

**Comment by patrickxb** [comment]
Is there another PR with JS changes to implement the new LoginUI functions?

**Comment by pzduniak** [comment]
there were like 2 or 3 lines changed in the frontend, the "display" stuff isnt handled by the frontend at all

**Comment by jakob223** [comment]
I think it would be good if we eventually moved the frontend over to use the same "display" RPCs as the CLI, so we could change the process later more easily if we wanted to.

**Pull request merged**